### PR TITLE
Remove useless comments

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: NewRelic/BlackholeInteractor.php [hunk 1]
```diff
@@ -13,9 +13,6 @@
 
 namespace Ekino\Bundle\NewRelicBundle\NewRelic;
 
-/**
- * Do not log anything.
```
Comment (Nyholm) [inline NewRelic/BlackholeInteractor.php]: This should no be removed.

File: Listener/ExceptionListener.php [hunk 2]
```diff
@@ -15,27 +15,15 @@
 use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
 use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
 
-/**
- * Notices the interactor on exception.
```
Comment (Nyholm) [inline Listener/ExceptionListener.php]: This has some value.

Comment (jderusse) [reply Listener/ExceptionListener.php]: Comment reworded

File: NewRelic/BlackholeInteractor.php (hunk 1 above)
Comment (jderusse) [reply NewRelic/BlackholeInteractor.php]: Comment reworded

**Pull request closed without merging**